python-cell.el

```diff
@@ -151,7 +155,9 @@ the command `python-cell-mode' to turn Python-Cell mode on."
     ;; (goto-char end)
     ;; (push-mark start)
     ;; (activate-mark)))
-    (python-shell-send-region start end)))
+    (pcase python-cell-backend
+      ('shell (python-shell-send-region start end))
+      ('jupyter (jupyter-eval-region start end)))))
```

Comment:
I guess that the byte-compiler will spit out a warning when emacs-juypter is not installed. To fix this warning `jupyer-eval` needs to be declared in advance (see `declare-function`)

Reply:
This should be addressed via `6b0b2ac`